draco/analysis/transform.py

```diff
@@ -1534,6 +1534,26 @@ class Jackknife(MixData):
     require_nonzero_weight = config.Property(proptype=bool, default=True)
 
 
+class MixTwoDatasets(MixData):
```

Comment:
If we are adding an HI simulation to the daily data during the quarterstacking stage, we will iterate over days, mix in the simulation, estimate/correct the gains, and then add the mixed data to the quarterstack.  I think the mixed data needs to be returned as an output of `process`, but here it looks like it will be returned as an output of `process_finish`. If it's output at `process_finish`, will the pipeline then loop back and start the next day in the list?

Reply:
Oh yeah, you're right, it will have to move to `process`

Reply:
And will `process_finish` be called for each day, or will the pipeline add all of the days to the mix before finally calling `process_finish`?

Reply:
I've made some changes which should address this